pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -314,10 +322,10 @@ func getMonitoringENV() []corev1.ServicePort {
 	}
 }
 
-// newRunnerPodForCR defines secondary resource #1 in same namespace as CR */
+// newRunnerPodForCR defines secondary resource #1 in same namespace as CR
 func newRunnerPodForCR(ce chaosTypes.EngineInfo) (*corev1.Pod, error) {
-	if len(ce.AppExperiments) == 0 || ce.AppUUID == "" {
-		return nil, errors.New("expected aExList not found")
+	if (len(ce.AppExperiments) == 0 || ce.AppUUID == "") && ce.Instance.Spec.ChaosType == "app" {
```

Comment:
Not require to check `chaosType`

Reply:
We are checking for `chaosType==app` because In case of infra chaos the AppUUID will be empty so it will fail
 Note: we are updating AppUUID inside annotation check and we are skipping annotation for infra chaos

Reply:
okay